src/main/java/com/uber/jenkins/phabricator/CommentBuilder.java

```diff
@@ -82,6 +83,7 @@ public void processParentCoverage(CodeCoverageMetrics parentCoverage, String bas
         String lineCoverageDisplay = String.format("%.3f", lineCoveragePercent);
 
         if (coverageDelta > 0) {
+            passCoverage = false;
```

Comment:
This seems... wrong? If there's no parent coverage, then this returns true, but if it increases, it returns false?